.github/actions/ortdocker/action.yml

```diff
@@ -43,8 +43,8 @@ runs:
     - name: Install Python
       uses: actions/setup-python@v4
       with:
-        python-version: '3.10'
-        cache: 'pip'
+        python-version: "3.10"
+        cache: "pip"
```

Comment:
Please also don't mix-in other unrelated changes like this changing of quotes.

Reply:
> or change all to double quote, or to single quote

I have no problem with that, but this should be done in a separate commit consistently for all files then.

Reply:
The name change and the fix are coupled, do it in two steps will not benefit at all. So the pr is done, just waiting for review.

Reply:
> The name change and the fix are coupled

Changing the quoting style is not coupled to renaming images.

Reply:
No, the hashing change not the quotes. quotes will be done other time as you suggested.

Reply:
Is weird that this is threaded over the hashing changes. Sorry for the confusion.